Dockerfile

```diff
@@ -90,9 +101,10 @@ LABEL name=${BIN_NAME}\
       description="Consul dataplane manages the proxy that runs within the data plane layer of Consul Service Mesh."
 
 COPY --from=go-discover /go/bin/discover /usr/local/bin/
-COPY --from=envoy-fips-binary /usr/local/bin/envoy /usr/local/bin/
+COPY --from=setcap-envoy-fips-binary /usr/local/bin/envoy /usr/local/bin/
```

Comment:
Notably this now copies the fips version of envoy that **has** had `setcap` run on it

Reply:
@skpratt is there any specific testing I should do around this fips image?

Reply:
Nope, it should be treated the same way as the non-FIPS one.